nmaipy/parcels.py

```diff
@@ -116,13 +116,21 @@
     },
 }
 
-TREE_BUFFERS_M = dict(
+BUFFER_ZONES_M = dict(
+    buffer_0ft=0.0,
     buffer_5ft=1.524,
     buffer_10ft=3.048,
     buffer_30ft=9.144,
     buffer_100ft=30.48,
 )
 
+BUFFER_CLASSES = {
+    "tree": VEG_MEDHIGH_ID,
```

Comment:
Suggest removing "tree" as it is not in our filings and should not be included in the unioned area for each zone.

Reply:
We can remove - but I thought it might be useful for some cases. For the full union, "tree" should be a strict subset of "woody composite", so shouldn't actually change it (woody composite is tree+woody I believe?).

Reply:
Ah, in that case it's fine to keep it since it doesn't add any additional area.